src/main/kotlin/nl/info/client/keycloak/KeycloakEmployeesAdminClientConfiguration.kt

```diff
@@ -35,7 +34,7 @@ class KeycloakEmployeesAdminClientConfiguration @Inject constructor(
     private val realmName: String,
 ) {
     companion object {
-        private val LOG = Logger.getLogger(IdentityService::class.java.name)
+        private val LOG = Logger.getLogger(KeycloakEmployeesAdminClientConfiguration::class.java.name)
```

Comment:
Can't we use `KotlinLogging`?

Reply:
We should, but I don't want to do it with this PR

Reply:
We use Java logging everywhere. Switching to a Kotlin logging should be done separately indeed and we should probably discuss it first and have a story for it.

Reply:
Personally what I had in mind was: move all Java code to Kotlin first and only then change the logging system.

Reply:
check, right you are...